examples/IBFE/explicit/ex5/input2d.iim

```diff
@@ -0,0 +1,247 @@
+// physical parameters
+Re = 200.0
+MU = 1.0/Re
+RHO = 1.0
+R = 0.5        											// radius of the cylinder
+U_MAX = 2.0
+// grid spacing parameters
+L = 60.0                                       // width of computational domain
+MAX_LEVELS = 6                                 // maximum number of levels in locally refined grid
+REF_RATIO  = 2                                 // refinement ratio between levels
+N = 64                                         // actual    number of grid cells on coarsest grid level
+NFINEST = (REF_RATIO^(MAX_LEVELS - 1))*N       // effective number of grid cells on finest   grid level
+DX0 = L/N                                      // mesh width on coarsest grid level
+DX  = L/NFINEST                                // mesh width on finest   grid level
+MFAC = 2.0                                     // ratio of Lagrangian mesh width to Cartesian mesh width
+ELEM_TYPE = "TRI3"                             // type of element to use for structure discretization
+PK1_DEV_QUAD_ORDER = "SEVENTH"
+PK1_DIL_QUAD_ORDER = "FIFTH"
+USE_BOUNDARY_MESH = TRUE
+
+// solver parameters
+IB_DELTA_FUNCTION          = "IB_3"            // the type of smoothed delta function to use for Lagrangian-Eulerian interaction
+SPLIT_FORCES               = TRUE             // whether to split interior and boundary forces
+USE_PRESSURE_JUMP_CONDITIONS        = TRUE             // whether to impose pressure jumps at fluid-structure interfaces
+USE_VELOCITY_JUMP_CONDITIONS = TRUE			   // whether to impose the velocity jumps at fluid-structure interfaces
+COMPUTE_FLUID_TRACTION 	   = TRUE              // whether to compute the exterior fluid traction
+USE_CONSISTENT_MASS_MATRIX = TRUE              // whether to use a consistent or lumped mass matrix
+
+USE_L2_LAGRANGE_FAMILY = FALSE  				// Whether to use L2_Lagrange family for the calc of jump variables and traction
+
+WSS_CALC_WIDTH 				= 1.05			   // ratio multiplied by the diagonal side of the grid spacing to calculate wss
+P_CALC_WIDTH 			   = 1.5               // ratio multiplied by the diogonal side of the grid spacing to calculate the pressure
```

Comment:
```suggestion
P_CALC_WIDTH 			   = 1.5               // ratio multiplied by the diagonal side of the grid spacing to calculate the pressure
```

Reply:
@boyceg should we keep these values in the input files? WSS_CALC_WIDTH = 1.05 and P_CALC_WIDTH = 1.5 have worked for all cases, but maybe we can do a little more investigation further along?

Reply:
I would suggest setting up the IIM input files for this example to correspond to what was done for the paper.